src/olympia/addons/tests/test_views.py

```diff
@@ -5563,9 +5593,15 @@ def test_update(self):
         )
         data = {'caption': {'en-US': 'a thing', 'fr': 'un thíng'}, 'position': 1}
 
+        # can't patch if not authenticated
         response = self.client.patch(url, data=data)
         assert response.status_code == 401
 
+        # can't patch if not your add-on
```

Comment:
These assertions don't seem to be related to the patch. Are they just things that you noticed were missing, or am I misunderstanding? Also, should there be tests that previews for themes cannot be updated, or is that pointless because they cannot be created?

Reply:
I should have added them earlier. They're kind of redundant because it's the same auth+permission setup used for Addon and other sub-Addon views, so there's no reason it wouldn't work, but it should be tested to be sure.